src/components/alert/alert.stories.mdx

```diff
@@ -46,6 +105,8 @@ When `dismissable` is `true`, a close button will display to the right of the me
   </Story>
 </Canvas>
 
+<ArgsTable story="Dismissable" />
```

Comment:
I usually add one `ArgsTable` in docs view. I can't decide if I like having them for each story or if it's just noise. Would be curious what others think.

Reply:
@tylersticka Fair! I also wondered the same, the argument for keeping them that I made to myself was that it's nice to see the configuration of each story. 

If I were scanning the docs, I might think...

"Oh, this flavor of Alert is the one I need!"

"Oh, cool, I can see how it is configured so I can make sure I configure it in the same way" or "Nice, I can see how this is configured, I just want to tweak this one detail"

(tweaks option)

"Perfect! Now I know how to configure what I need"

> Would be curious what others think.

Me too. 🙂

Reply:
I don't mind it too much

Reply:
@gerardo-rodriguez I would leave the `ArgsTable` for the first (basic) example.